app/scripts/views/coppa/coppa-age-input.js

```diff
@@ -56,6 +56,22 @@ define(function (require, exports, module) {
       age.val(age.val().substr(0, 3));
     },
 
+    onKeyDown: function (event) {
+      // helper function to check for digit
+      function isKeyADigit (keyCode) {
+        return ((keyCode == 8) || // backspace
```

Comment:
Please use `===` for equality checks, JavaScript will try to do coerce mismatched types to be the same type otherwise.